File: code/DatabaseEFC/DatabaseEFC/DTO/Administrator.cs
using System.ComponentModel.DataAnnotations;

namespace DatabaseEFC.DTO;

/// <summary>
/// Used to communicate via ReST
/// </summary>
public class Administrator
{
    /// <summary>
    /// The id assigned to the administrator. Sent as string to not lose precision
    /// </summary>
    public string? AdministratorId { get; set; }
    
    /// <summary>
    /// The volunteer id of the administrator. Sent as string to not lose precision
    /// </summary>
    [Required]
    public string VolunteerId { get; set; }
    /// <summary>
    /// The id of the volunteer's manager role. Sent as string to not lose precision
    /// </summary>
    public string? ManagerId { get; set; }
}

File: code/java-tier2/src/main/java/dk/dsrhorsens/volunteers/service/user/rest/Administrator.java
package dk.dsrhorsens.volunteers.service.user.rest;

/**
 * Administrator DTO class
 */
public class Administrator {
    private String administratorId;
    private String volunteerId;
    private String managerId;

    public void setAdministratorId(String administratorId) {
        this.administratorId = administratorId;
    }

    public void setVolunteerId(String volunteerId) {
        this.volunteerId = volunteerId;
    }

    public void setManagerId(String managerId) {
        this.managerId = managerId;
    }
}
